README.md

```diff
@@ -65,10 +65,77 @@ You do not need to set the description, that just makes it easier for help comma
 
 You can also override the method String[] getAliases() this allows you to set an alias for the command for example `p` instead of `ping`.
 
+# Setting up MySQL
+For ease of use, JBA comes built in with a few classes to help you use MySQL as a database system. You can set up the database for usage in the pre-generated `public void run()` method made on implementing the JBA class. Inside that run method, simply enter the code below, replacing each String with your details for your MySQL DB.
+
+```java
+setupMySQL("mysql_user", "mysql_pass", "mysql_address", "mysql_dbname");
+```
+
+This can be easily configured with the Config system in JBA, which you can scroll further down to see.
```

Comment:
Link don't say scroll down, linked is the same as Discord markdown. [text](#header)